packages/blade/src/components/Checkbox/CheckboxIcon/CheckboxIconWrapperStyles.ts

```diff
@@ -0,0 +1,83 @@
+import type { CSSObject } from 'styled-components';
```

Comment:
why not name it as `styles.ts` since you're following that same convention above as well for checkboxgroup?

Reply:
I suppose we can. There's just one component thats why.

Reply:
let's follow the same convention atleast in the same contextual flow? :D

Reply:
We deleted the style.ts file. So lets keep it this way.